packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -1074,7 +1079,16 @@ class RawEditorState extends EditorState
     super.dispose();
   }
 
+  // Storing the raw text of last textEditingValue
```

Comment:
I'm not sure why we need this. If `TextEditingValue.text` has not changed, then it means `TextEditingValue.selection` has changed, shouldn't this be capture by `_handleSelectionChanges`?

Reply:
Well, I don't want to detect when selection changed. Here I'm interested in text changes and the `_didChangeTextEditingValue` gets called on controller updates which happens on both text or selection changes. @amantoux

Reply:
My bad again, reviewing this PR on holiday with my kids :)

Why not handle this in `RawEditorStateTextInputClientMixin::updateEditingValueWithDeltas` by adding
```dart
if (data.isNotEmpty) hideToolbar(false);
```

Reply:
We can do that. But it doesn't get called when changes come from controller. I'm not sure which one is the correct behavior. WDYT? @amantoux

Reply:
Yes I understand but I think controller-based changes should be programmatic and shouldn't affect how the editor prompts the toolbar to the user.
Do you agree?

Reply:
Agreed. Will change that. @amantoux